usr/share/laptop-mode-tools/modules/runtime-pm

```diff
@@ -123,8 +134,8 @@ if [ x$CONTROL_RUNTIME_AUTOSUSPEND = x1 ] || [ x$ENABLE_AUTO_MODULES = x1 -a x$C
 				fi
 
 				if [ x$USE_DEVICE = x1 ]; then
-					if [ -f "$runtime_device"/power/autosuspend ]; then
-						echo $AUTOSUSPEND_TIMEOUT > "$runtime_device"/power/autosuspend;
+					if [ -f "$runtime_device/$autosuspend_path" ]; then
```

Comment:
Hello @IyadKandalaft. Thank you for the report and the PR. And sorry for the delay in my response. Lots of things in personal life has demanded my attention.


Instead of the version string comparison, can't we just check for the paths ?
This will easily serve as fallback for all old kernel interfaces. Personally, I'm trying my best to avoid string comparisons.

if -f power/autosuspend_delay_ms
   do_something();
elif -f power/autosuspend
   do_alternate();
else
   bail_out();
fi

Reply:
I think that in one minor version (possibly 2.6.37), the autosuspend_delay_ms was there but didn't do anything.  Anyways, that's one minor version and it's cleaner to just check if auto suspend delay ms is present after checking for autosuspend.

Reply:
So do we still need this PR ? Or can we close it now ?

Reply:
It can be closed.